pkg/stratus/providers.go

```diff
@@ -22,6 +22,12 @@ func EnsureAuthenticated(platform Platform) error {
 				"Make sure you are authenticated against AWS, and you have a default region set in your AWS config " +
 				"or environment (export AWS_DEFAULT_REGION=us-east-1)")
 		}
+	case Azure:
```

Comment:
_Note: I realize this isn't part of the PR, it's code that I moved in a previous commit independently of your PR but that got lost in the git void_